spec/support/compare_locales_support.rb

```diff
@@ -1,48 +1,8 @@
 # frozen_string_literal: true
 module CompareLocalesSupport
   LOCALES_DIR = './config/locales'
-  LOCALES_FILES = {
-    english: "#{LOCALES_DIR}/en.yml",
-    spanish: "#{LOCALES_DIR}/es.yml",
-    portuguese: "#{LOCALES_DIR}/pt-BR.yml",
-    swedish: "#{LOCALES_DIR}/sv.yml",
-    norwegian: "#{LOCALES_DIR}/nb.yml",
-    dutch: "#{LOCALES_DIR}/nl.yml",
-    italian: "#{LOCALES_DIR}/it.yml",
-    vietnamese: "#{LOCALES_DIR}/vi.yml",
-    german: "#{LOCALES_DIR}/de.yml",
-    french: "#{LOCALES_DIR}/fr.yml",
-    devise_english: "#{LOCALES_DIR}/devise.en.yml",
-    devise_spanish: "#{LOCALES_DIR}/devise.es.yml",
-    devise_portuguese: "#{LOCALES_DIR}/devise.pt-BR.yml",
-    devise_swedish: "#{LOCALES_DIR}/devise.sv.yml",
-    devise_norwegian: "#{LOCALES_DIR}/devise.nb.yml",
-    devise_dutch: "#{LOCALES_DIR}/devise.nl.yml",
-    devise_italian: "#{LOCALES_DIR}/devise.it.yml",
-    devise_vietnamese: "#{LOCALES_DIR}/devise.vi.yml",
-    devise_german: "#{LOCALES_DIR}/devise.de.yml",
-    devise_french: "#{LOCALES_DIR}/devise.fr.yml",
-    devise_invitable_english: "#{LOCALES_DIR}/devise_invitable.en.yml",
-    devise_invitable_spanish: "#{LOCALES_DIR}/devise_invitable.es.yml",
-    devise_invitable_portuguese: "#{LOCALES_DIR}/devise_invitable.pt-BR.yml",
-    devise_invitable_swedish: "#{LOCALES_DIR}/devise_invitable.sv.yml",
-    devise_invitable_norwegian: "#{LOCALES_DIR}/devise_invitable.nb.yml",
-    devise_invitable_dutch: "#{LOCALES_DIR}/devise_invitable.nl.yml",
-    devise_invitable_italian: "#{LOCALES_DIR}/devise_invitable.it.yml",
-    devise_invitable_vietnamese: "#{LOCALES_DIR}/devise_invitable.vi.yml",
-    devise_invitable_german: "#{LOCALES_DIR}/devise_invitable.de.yml",
-    devise_invitable_french: "#{LOCALES_DIR}/devise_invitable.fr.yml",
-    kaminari_english: "#{LOCALES_DIR}/kaminari.en.yml",
-    kaminari_spanish: "#{LOCALES_DIR}/kaminari.es.yml",
-    kaminari_portuguese: "#{LOCALES_DIR}/kaminari.pt-BR.yml",
-    kaminari_swedish: "#{LOCALES_DIR}/kaminari.sv.yml",
-    kaminari_norwegian: "#{LOCALES_DIR}/kaminari.nb.yml",
-    kaminari_dutch: "#{LOCALES_DIR}/kaminari.nl.yml",
-    kaminari_italian: "#{LOCALES_DIR}/kaminari.it.yml",
-    kaminari_vietnamese: "#{LOCALES_DIR}/kaminari.vi.yml",
-    kaminari_german: "#{LOCALES_DIR}/kaminari.de.yml",
-    kaminari_french: "#{LOCALES_DIR}/kaminari.fr.yml"
-  }.freeze
+  ENGLISH_FILES = Dir.glob("#{LOCALES_DIR}/**/*en.yml")
+  NON_ENGLISH_LOCALES = ['es', 'de', 'it', 'nb', 'nl', 'pt-BR', 'sv', 'vi', 'fr']
```

Comment:
I'm wondering if there is a way to reference the list of locales that is defined in in [config/application.rb](https://github.com/ifmeorg/ifme/blob/003785f3d72587f5ae6925d69e94a155b46f0b03/config/application.rb#L53), as I think that would be better but I've not yet figured out how to do that.